{
  "exploitdb": [
    {
      "cve": "CVE-2019-19844",
      "id": "47879",
      "file": "exploits/python/webapps/47879.md",
      "description": "Django \u003c 3.0 \u003c 2.2 \u003c 1.11 - Account Hijack",
      "author": "Ryuji Tsutsui",
      "application_url": "",
      "source_url": "https://ryu22e.org/en/posts/2019/12/25/django-cve-2019-19844/",
      "screenshot": "",
      "verified": true,
      "codes": "CVE-2019-19844",
      "tags": "",
      "type": "webapps",
      "platform": "python",
      "port": "",
      "aliases": "",
      "date_added": "2020-01-06T00:00:00Z",
      "date_updated": "2020-04-13T00:00:00Z",
      "date_published": "2019-12-24T00:00:00Z"
    }
  ],
  "itw": [
    {
      "id": "CVE-2019-19844",
      "description": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2019-19844",
      "reportURL": "https://github.com/0xsha/CVE_2019_19844",
      "timeStamp": "2020-01-18T13:32:21Z"
    },
    {
      "id": "CVE-2019-19844",
      "description": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2019-19844",
      "reportURL": "https://github.com/andripwn/django_cve201919844",
      "timeStamp": "2019-12-25T10:05:48Z"
    },
    {
      "id": "CVE-2019-19844",
      "description": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2019-19844",
      "reportURL": "https://github.com/ryu22e/django_cve_2019_19844_poc",
      "timeStamp": "2019-12-21T04:30:34Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2019-19844",
      "url": "https://github.com/Mr-xn/Penetration_Testing_POC",
      "added_at": "2022-02-10T19:32:13Z"
    },
    {
      "id": "CVE-2019-19844",
      "url": "https://packetstormsecurity.com/files/155872/Django-Account-Hijack.html",
      "added_at": "2022-02-10T19:32:13Z"
    }
  ],
  "nomisec": [
    {
      "cveId": "CVE-2019-19844",
      "id": 234739158,
      "name": "CVE_2019_19844",
      "full_name": "0xsha/CVE_2019_19844",
      "owner": {
        "login": "0xsha",
        "id": 54356171,
        "avatar_url": "https://avatars.githubusercontent.com/u/54356171?v=4",
        "html_url": "https://github.com/0xsha",
        "user_view_type": "public"
      },
      "html_url": "https://github.com/0xsha/CVE_2019_19844",
      "description": "CVE-2019-19844 Docker Edition",
      "fork": false,
      "created_at": "2020-01-18T13:32:21Z",
      "updated_at": "2025-04-08T21:47:33Z",
      "pushed_at": "2021-09-22T18:25:53Z",
      "stargazers_count": 3,
      "watchers_count": 3,
      "has_discussions": false,
      "forks_count": 3,
      "allow_forking": true,
      "is_template": false,
      "web_commit_signoff_required": false,
      "topics": [],
      "visibility": "public",
      "forks": 3,
      "watchers": 3,
      "score": 0,
      "subscribers_count": 2
    },
    {
      "cveId": "CVE-2019-19844",
      "id": 230083007,
      "name": "django_cve201919844",
      "full_name": "andripwn/django_cve201919844",
      "owner": {
        "login": "andripwn",
        "id": 52893492,
        "avatar_url": "https://avatars.githubusercontent.com/u/52893492?v=4",
        "html_url": "https://github.com/andripwn",
        "user_view_type": "public"
      },
      "html_url": "https://github.com/andripwn/django_cve201919844",
      "description": "PoC for CVE-2019-19844 ( https://www.djangoproject.com/weblog/2019/dec/18/security-releases/ )",
      "fork": false,
      "created_at": "2019-12-25T10:05:48Z",
      "updated_at": "2024-08-12T19:56:01Z",
      "pushed_at": "2020-09-20T16:23:45Z",
      "stargazers_count": 8,
      "watchers_count": 8,
      "has_discussions": false,
      "forks_count": 2,
      "allow_forking": true,
      "is_template": false,
      "web_commit_signoff_required": false,
      "topics": [],
      "visibility": "public",
      "forks": 2,
      "watchers": 8,
      "score": 0,
      "subscribers_count": 1
    },
    {
      "cveId": "CVE-2019-19844",
      "id": 229376107,
      "name": "django_cve_2019_19844_poc",
      "full_name": "ryu22e/django_cve_2019_19844_poc",
      "owner": {
        "login": "ryu22e",
        "id": 301892,
        "avatar_url": "https://avatars.githubusercontent.com/u/301892?v=4",
        "html_url": "https://github.com/ryu22e",
        "user_view_type": "public"
      },
      "html_url": "https://github.com/ryu22e/django_cve_2019_19844_poc",
      "description": "PoC for CVE-2019-19844(https://www.djangoproject.com/weblog/2019/dec/18/security-releases/)",
      "fork": false,
      "created_at": "2019-12-21T04:30:34Z",
      "updated_at": "2025-04-15T05:38:30Z",
      "pushed_at": "2019-12-24T12:31:47Z",
      "stargazers_count": 101,
      "watchers_count": 101,
      "has_discussions": false,
      "forks_count": 12,
      "allow_forking": true,
      "is_template": false,
      "web_commit_signoff_required": false,
      "topics": [
        "cve-2019-19844",
        "django",
        "python",
        "security",
        "vulnerability"
      ],
      "visibility": "public",
      "forks": 12,
      "watchers": 101,
      "score": 0,
      "subscribers_count": 3
    }
  ],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2019-19844",
      "url": "https://github.com/0xsha/CVE_2019_19844",
      "added_at": "2020-01-18T13:32:21Z",
      "score": 0.43000000000000005,
      "template_for": ""
    },
    {
      "cve": "CVE-2019-19844",
      "url": "https://github.com/Mr-xn/Penetration_Testing_POC",
      "added_at": "2022-02-10T19:32:13Z",
      "score": 0.9,
      "template_for": ""
    },
    {
      "cve": "CVE-2019-19844",
      "url": "https://github.com/andripwn/django_cve201919844",
      "added_at": "2019-12-25T10:05:48Z",
      "score": 0.48000000000000004,
      "template_for": ""
    },
    {
      "cve": "CVE-2019-19844",
      "url": "https://github.com/ryu22e/django_cve_2019_19844_poc",
      "added_at": "2019-12-21T04:30:34Z",
      "score": 1,
      "template_for": ""
    },
    {
      "cve": "CVE-2019-19844",
      "url": "https://packetstormsecurity.com/files/155872/Django-Account-Hijack.html",
      "added_at": "2022-02-10T19:32:13Z",
      "score": 0.4,
      "template_for": ""
    },
    {
      "cve": "CVE-2019-19844",
      "url": "https://www.exploit-db.com/exploits/47879",
      "added_at": "2019-12-24T00:00:00Z",
      "score": 1,
      "template_for": ""
    }
  ]
}
